v2/internal/pkg/clusterresources/clusterresources.go

```diff
@@ -169,6 +174,10 @@ func writeMirrorSet[T confv1.ImageDigestMirrorSet | confv1.ImageTagMirrorSet](mi
 }
 
 func (o *ClusterResourcesGenerator) CatalogSourceGenerator(allRelatedImages []v2alpha1.CopyImageSchema) error {
+	if len(o.Config.Mirror.Operators) == 0 {
+		o.Log.Info("📄 No operators mirrored. Skipping CatalogSource file generation.")
+	}
+
 	o.Log.Info("📄 Generating CatalogSource file...")
```

Comment:
Printing this message after saying "No operators mirrored" could cause confusion. 

It would be possible to add a return statement after line 178.

Reply:
return statement added